src/components/password_modal/PasswordModal.jsx

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import {
+  Container,
+  Button,
+  FormGroup,
+  FormLabel,
+  Modal,
+} from "react-bootstrap";
+import { Formik, Form, Field } from "formik";
+import * as Yup from "yup";
+
+function PasswordModal(props) {
+  const { isOpen, onCloseModal } = props;
```

Comment:
you can deconstruct the props in the function parameter, i believe like this "function PasswordModal({ isOpen, onCloseModal }) {..."